tests/test_mlkit.py

```diff
@@ -763,7 +763,12 @@ def teardown_class(cls):
         testutils.cleanup_apps()
 
     @staticmethod
-    def _url(project_id, model_id):
+    def _update_url(project_id, model_id):
+        update_url = 'projects/{0}/models/{1}?updateMask=state.published'
+        return BASE_URL + update_url.format(project_id, model_id)
```

Comment:
Nit: Consider doing the `format()` on the previous line.

Reply:
it's one character too long. I can wrap it or I can leave it? Which do you prefer?

Reply:
May be something like:

```
update_url = 'projects/{0}/models/{1}?updateMask=state.published'.format(
    project_id, model_id)
```